vidarr-pluginapi/src/main/java/ca/on/oicr/gsi/vidarr/ConsumableResource.java

```diff
@@ -46,4 +90,11 @@
    */
   ConsumableResourceResponse request(
       String workflowName, String workflowVersion, String vidarrId, Optional<JsonNode> input);
+
+  /**
+   * Called to initialise this workflow engine.
```

Comment:
Won't most of these be something other than workflow engines?

Reply:
Yes. I did some copy pasta.